app/controllers/user_sessions_controller.rb

```diff
@@ -77,15 +77,8 @@ def handle_social_login_flow(auth)
             return_to = session[:openid_return_to]
             session[:openid_return_to] = nil
             redirect_to return_to + hash_params
-          elsif params[:return_to] && params[:return_to].split('/')[0..3] == ["", "subscribe", "multiple", "tag"]
```

Comment:
why have you removed it? This was a feature I built. Any special reasons?

Reply:
I removed it because in the issue and when we chatted about oauth, you said when users sign up they should be redirected to the dashboard. This is the code that handles that part:
https://github.com/publiclab/plots2/blob/c4f849d0c9f3a5d54ad9e0ebd6b433ab42dc47ae/app/controllers/user_sessions_controller.rb#L66-L82

So as you can see, if a user without an account goes to `/subscribe/multiple/tag/water` and then they are prompted to sign up, when they signup they will be redirected to the subscriptions instead of the dashboard.

Reply:
I think this part should be discussed separately. Is this needed in this pr?
Title says LOGIN. Isn't it?
We intentionally created this feature due to some reason. Can you please open up an issue for discussion and remove these changes from here if they don't interfere with the OAuth changes?

Reply:
Yeah no problem

Reply:
Sure